src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        StringBuilder builder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(" ");
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException();
```

lets add some informative exception message here([checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#dont-ignore-exceptions): Don’t ignore exceptions)